pkg/volume/glusterfs/glusterfs.go

```diff
@@ -1037,7 +1045,17 @@ func parseClassParameters(params map[string]string, kubeClient clientset.Interfa
 			if len(v) != 0 {
 				parseThinPoolSnapFactor = v
 			}
-
+		case "customepnameprefix":
+			// If the string has > 'maxCustomEpNamePrefixLen' chars, the final endpoint name will
+			// exceed the limitation of 63 chars, so fail if prefix is > 'maxCustomEpNamePrefixLen'
+			// characters. This is only applicable for 'customepnameprefix' string and default ep name
+			// string will always pass.
+			if len(v) < maxCustomEpNamePrefixLen {
+				cfg.customEpNamePrefix = v
+			} else {
+				return nil, fmt.Errorf("'customepnameprefix' value should be < %d characters", maxCustomEpNamePrefixLen)
+			}
+			cfg.customEpNamePrefix = v
```

Comment:
The same line is already on line 1054  (in `if` branch).

Reply:
ah. removed.